controller_manager/src/controller_manager.cpp

```diff
@@ -293,6 +293,14 @@ ControllerManager::ControllerManager(
 
 void ControllerManager::init_controller_manager()
 {
+  controller_manager_activity_publisher_ =
+    create_publisher<controller_manager_msgs::msg::ControllerManagerActivity>(
+      "~/activity", rclcpp::QoS(1).transient_local());
```

Comment:
The default QOS is reliable but it is better to be explicit here. So rclcpp::QOS(1).reliable().transient_local();

Reply:
The robot_state_publisher also uses the same approach: https://github.com/ros/robot_state_publisher/blob/rolling/src/robot_state_publisher.cpp#L135

Do you still think we should be explicit here?

Reply:
I think it is a good practice to be explicit and not make assumptions about default QoS. Just because it is done one way elsewhere doesn't make it the right thing to do. Having it explicit makes it more clearer to the user as well.

Reply:
👍🏾 . I'll make the change

Reply:
Done!